From ff0185597afb589f15e6b160ae2f022542b451d9 Mon Sep 17 00:00:00 2001
From: Christopher Feghali <christopher.feghali.xb@renesas.com>
Date: Thu, 30 Apr 2020 15:30:29 +0100
Subject: [PATCH] remove libraries and library dirs from cppextension

Not removing these lines breaks TorchVision as these libraries aren't
required.

Signed-off-by: Christopher Feghali <christopher.feghali.xb@renesas.com>
---
 torch/utils/cpp_extension.py | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/torch/utils/cpp_extension.py b/torch/utils/cpp_extension.py
index f8137da712..bf1820e88f 100644
--- a/torch/utils/cpp_extension.py
+++ b/torch/utils/cpp_extension.py
@@ -658,17 +658,6 @@ def CppExtension(name, sources, *args, **kwargs):
     include_dirs += include_paths()
     kwargs['include_dirs'] = include_dirs
 
-    library_dirs = kwargs.get('library_dirs', [])
-    library_dirs += library_paths()
-    kwargs['library_dirs'] = library_dirs
-
-    libraries = kwargs.get('libraries', [])
-    libraries.append('c10')
-    libraries.append('torch')
-    libraries.append('torch_cpu')
-    libraries.append('torch_python')
-    kwargs['libraries'] = libraries
-
     kwargs['language'] = 'c++'
     return setuptools.Extension(name, sources, *args, **kwargs)
 
-- 
2.17.1

